### ISSUE TITLE
Accessibility: Insufficient contrast on focused navigation links (mobile)

### ISSUE THREAD

**Issue by aduth** [GSA/centers-of-excellence#370]
For mobile users, or users who zoom the page to the point that mobile styles activate, focused navigation links which are contained within an expanded accordion parent are styled with an insufficient contrast of 2.1:1, failing to meet WCAG Level AA minimum of 4.5:1.

https://webaim.org/resources/contrastchecker/?fcolor=212121&bcolor=205493

**Screenshots:**

![mobile menu focus contrast footer](https://user-images.githubusercontent.com/1779930/86289155-2e65f880-bbb9-11ea-9ce1-787ca38695fd.png)

![mobile menu focus contrast](https://user-images.githubusercontent.com/1779930/86289153-2e65f880-bbb9-11ea-86d9-4ba29591b549.png)

**Steps to Reproduce:**

1. Navigate to https://coe.gsa.gov/
2. Shrink the page or zoom to the point that mobile styles activate
3. Tab to the "Menu" button
4. Click the "Menu" button by pressing <kbd>Space</kbd>
5. <kbd>Tab</kbd> to "About Us"
6. Expand the collapsed menu items by pressing <kbd>Space</kbd>
7. <kbd>Tab</kbd> to the next menu item

_Expected result:_ Menu items are shown with sufficient contrast.
_Actual result:_ There is insufficient contrast of the text on the background.

**WCAG Success Criteria (AA)**: [1.4.3: Contrast (Minimum)](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html)

**Relevant Technique:** [General G18: Ensuring that a contrast ratio of at least 4.5:1 exists between text (and images of text) and background behind the text](https://www.w3.org/WAI/WCAG21/Techniques/general/G18)

**Remediation Guidance:**

It's assumed this navigation styling is largely inherited from use of [U.S. Web Design System](https://designsystem.digital.gov/). USWDS (both v1 and v2) do not suffer from this problem, so it is possible it is an issue of:

- An outdated version of USWDS is being used.
- Improper usage of USWDS is preventing the styling from taking effect.
